cmd/goal/inspect.go

```diff
@@ -72,7 +72,7 @@ type inspectTransaction struct {
 type inspectTxnHeader struct {
```

Comment:
With this change it looks like you should be able to remove `inspectTxnHeader` and `inspectPaymentTxnFields` altogether

Reply:
I'm okay with that being a follow-on cleanup issue to get this big chunk of code in.

Reply:
Fair enough; we should just try to not lose track of it

Reply:
noted in #129